doc/urdf_srdf/urdf_srdf_tutorial.rst

```diff
@@ -95,3 +95,66 @@ The SRDF can also store fixed configurations of the robot. A typical example of
 SRDF Documentation
 ^^^^^^^^^^^^^^^^^^
 For information about the syntax for the SRDF, read more details on the `ROS SRDF Wiki page <http://www.ros.org/wiki/srdf>`_.
+
+Loading the URDF and SRDF
+-------------------------
+All the components of MoveIt that use the :moveit_core:`RobotModel` need to have access to the URDF and SRDF to function properly. In ROS 1, this was accomplished by loading the XML of each into a string parameter (``/robot_description`` and ``/robot_description_semantic`` respectively) into the global parameter server. ROS 2 does not have a global parameter server, so making sure all the appropriate nodes have access requires a little more work.
+
+Launch File Specification
+^^^^^^^^^^^^^^^^^^^^^^^^^
+One option is to set the parameters for each node that requires them, which is typically done using a launch file.
+
+Loading the URDF often uses xacro, and so loading it looks like
+
+
+.. code-block:: python
+
+    from launch_ros.parameter_descriptions import ParameterValue
+    from launch.substitutions import Command
+
+    robot_description = ParameterValue(Command(['xacro ', PATH_TO_URDF]),
+                                       value_type=str)
+
+Meanwhile, the SRDF must be read in explicitly.
+
+.. code-block:: python
+
+    with open(PATH_TO_SRDF, 'r') as f:
+        semantic_content = f.read()
+
+Then the values must be loaded into EACH node.
+
+.. code-block:: python
+
+    move_group_node = Node(package='moveit_ros_move_group', executable='move_group',
+                           output='screen',
+                           parameters=[{
+                                'robot_description': robot_description,
+                                'robot_description_semantic': semantic_content,
+                                # More params
+                           }],
+                           )
+
+String Topic Specification
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+The other approach is to use publish the two strings as topics. This pattern is already done with the `Robot State Publisher <https://github.com/ros/robot_state_publisher/blob/37aff2034b58794b78f1682c8fab4d609f5d2e29/src/robot_state_publisher.cpp#L136>`_ which publishes a ``std_msgs/msg/String`` message on the ``/robot_description`` topic. If that topic is already available, then there is no need to load the parameter explicitly in the launch file. The only place that the parameter needs to be explicitly loaded is in whatever is publishing that topic (e.g. Robot State Publisher).
```

Comment:
Would you mind also adding the launch commands to use robot_state_publisher or something to publish the robot_description?  What would you use if you didn't need a robot state publisher but still need something published to this topic because you are expecting to read from it (instead of from a parameter)?